mountSuspended does not render Options API's data and computed properties

### Environment

------------------------------
- Operating System: Darwin
- Node Version:     v18.19.0
- Nuxt Version:     3.12.4
- CLI Version:      3.13.2
- Nitro Version:    2.9.7
- Package Manager:  npm@10.2.3
- Builder:          -
- User Config:      compatibilityDate, devtools
- Runtime Modules:  -
- Build Modules:    -
------------------------------

### Reproduction

https://nuxttestutilsplayground-0esz--55067--134daa3c.local-credentialless.webcontainer.io

GitHub: https://github.com/inouetakuya/nuxt-test-utils-playground/pull/3

### Describe the bug

Here is the page component using the Options API.

```vue
<script lang="ts">  
export default defineNuxtComponent({  
  name: 'OptionsApiPage',  
  setup() {  
    return {  
      greetingInSetup: 'Hello, setup!',  
    };  
  },  
  async asyncData() {  
    return {  
      greetingInAsyncData: 'Hello, asyncData!',  
    };  
  },  
  data() {  
    return {  
      greetingInData: 'Hello, data!',  
    };  
  },  
  computed: {  
    greetingInComputed() {  
      return 'Hello, computed property!';  
    },  
  },  
});  
</script>  
  
<template>  
  <p data-testid="greeting-in-setup">greetingInSetup: {{ greetingInSetup }}</p>  
  <p data-testid="greeting-in-async-data">  
    greetingInAsyncData: {{ greetingInAsyncData }}  
  </p>  
  <p data-testid="greeting-in-data">greetingInData: {{ greetingInData }}</p>  
  <p data-testid="greeting-in-computed">  
    greetingInComputed: {{ greetingInComputed }}  
  </p>  
</template>
```

When testing this page component using the mountSuspended method, the data and computed properties of the Options API are not rendered.  

On the other hand, if the same page component is tested with the mount method, both the Options API data and the computed properties are rendered.

```ts
import { mountSuspended } from '@nuxt/test-utils/runtime';  
import { mount, flushPromises } from '@vue/test-utils';  
import OptionsApiPage from '~/pages/options-api.vue';  
  
describe('OptionsApiPage', () => {  
  let wrapper;  
  
  describe('Using mountSuspended', () => {  
    beforeEach(async () => {  
      wrapper = await mountSuspended(OptionsApiPage, {  
        route: '/options-api',  
      });  
    });  
  
    it('should render greeting in setup', () => {  
      expect(wrapper.find('[data-testid="greeting-in-setup"]').text()).toBe(  
        'greetingInSetup: Hello, setup!',  
      );  
    });  
  
    it('should render greeting in asyncData', () => {  
      expect(  
        wrapper.find('[data-testid="greeting-in-async-data"]').text(),  
      ).toBe('greetingInAsyncData: Hello, asyncData!');  
    });  
  
    it('should render greeting in data', () => {  
      expect(wrapper.find('[data-testid="greeting-in-data"]').text()).toBe(  
        'greetingInData: Hello, data!',  
      );  
    });  
  
    it('should render greeting in computed', () => {  
      expect(wrapper.find('[data-testid="greeting-in-computed"]').text()).toBe(  
        'greetingInComputed: Hello, computed property!',  
      );  
    });  
  });  
  
  describe('Using mount', () => {  
    beforeEach(async () => {  
      const component = defineComponent({  
        components: { OptionsApiPage },  
        template: '<Suspense><OptionsApiPage /></Suspense>',  
      });  
      wrapper = mount(component);  
      await flushPromises();  
    });  
  
    it('should render greeting in setup', () => {  
      expect(wrapper.find('[data-testid="greeting-in-setup"]').text()).toBe(  
        'greetingInSetup: Hello, setup!',  
      );  
    });  
  
    it('should render greeting in asyncData', () => {  
      expect(  
        wrapper.find('[data-testid="greeting-in-async-data"]').text(),  
      ).toBe('greetingInAsyncData: Hello, asyncData!');  
    });  
  
    it('should render greeting in data', () => {  
      expect(wrapper.find('[data-testid="greeting-in-data"]').text()).toBe(  
        'greetingInData: Hello, data!',  
      );  
    });  
  
    it('should render greeting in computed', () => {  
      expect(wrapper.find('[data-testid="greeting-in-computed"]').text()).toBe(  
        'greetingInComputed: Hello, computed property!',  
      );  
    });  
  });  
});
```

### Additional context

_No response_

### Logs

```shell-script
$ npm test

> test
> vitest run


 RUN  v2.0.5 /Users/inouetakuya/src/github.com/inouetakuya/nuxt-test-utils-playground

stdout | pages/script-setup.nuxt.spec.ts
<Suspense> is an experimental feature and its API will likely change.

stdout | pages/options-api.nuxt.spec.ts
<Suspense> is an experimental feature and its API will likely change.

 ❯ pages/options-api.nuxt.spec.ts (8)
   ❯ OptionsApiPage (8)
     ❯ Using mountSuspended (4)
       ✓ should render greeting in setup
       ✓ should render greeting in asyncData
       × should render greeting in data
       × should render greeting in computed
     ✓ Using mount (4)
 ✓ pages/script-setup.nuxt.spec.ts (1)
 ✓ utils/example-util.spec.ts (1)

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯ Failed Tests 2 ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯

 FAIL  pages/options-api.nuxt.spec.ts > OptionsApiPage > Using mountSuspended > should render greeting in data
AssertionError: expected 'greetingInData:' to be 'greetingInData: Hello, data!' // Object.is equality

Expected: "greetingInData: Hello, data!"
Received: "greetingInData:"

 ❯ pages/options-api.nuxt.spec.ts:28:71
     26|
     27|     it('should render greeting in data', () => {
     28|       expect(wrapper.find('[data-testid="greeting-in-data"]').text()).toBe(
       |                                                                       ^
     29|         'greetingInData: Hello, data!',
     30|       );

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/2]⎯

 FAIL  pages/options-api.nuxt.spec.ts > OptionsApiPage > Using mountSuspended > should render greeting in computed
AssertionError: expected 'greetingInComputed:' to be 'greetingInComputed: Hello, computed p…' // Object.is equality

Expected: "greetingInComputed: Hello, computed property!"
Received: "greetingInComputed:"

 ❯ pages/options-api.nuxt.spec.ts:34:75
     32|
     33|     it('should render greeting in computed', () => {
     34|       expect(wrapper.find('[data-testid="greeting-in-computed"]').text()).toBe(
       |                                                                           ^
     35|         'greetingInComputed: Hello, computed property!',
     36|       );

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[2/2]⎯

 Test Files  1 failed | 2 passed (3)
      Tests  2 failed | 8 passed (10)
   Start at  16:38:07
   Duration  740ms (transform 303ms, setup 513ms, collect 188ms, tests 41ms, environment 237ms, prepare 331ms)
```

--- a/examples/app-vitest-full/pages/other/options-api.vue
+++ b/examples/app-vitest-full/pages/other/options-api.vue
@@ -1,0 +1,57 @@
+<template>
+  <ul>
+    <li data-testid="greetingInSetup">{{ greetingInSetup }}</li>
+    <li data-testid="greetingInData1">{{ greetingInData1 }}</li>
+    <li data-testid="greetingInData2">{{ greetingInData2 }}</li>
+    <li data-testid="greetingInComputed">{{ greetingInComputed }}</li>
+    <li data-testid="computedData1">{{ computedData1 }}</li>
+    <li data-testid="computedGreetingInMethods">{{ computedGreetingInMethods }}</li>
+    <li data-testid="greetingInMethods">{{ greetingInMethods() }}</li>
+    <li data-testid="returnData1">{{ returnData1() }}</li>
+    <li data-testid="returnComputedData1">{{ returnComputedData1() }}</li>
+  </ul>
+</template>
+
+<script lang="ts">
+export default defineNuxtComponent({
+  name: 'OptionsApiPage',
+  setup() {
+    return {
+      greetingInSetup: 'Hello, setup',
+    }
+  },
+  async asyncData() {
+    return {
+      greetingInData2: 'Hello, overwritten by asyncData',
+    }
+  },
+  data() {
+    return {
+      greetingInData1: 'Hello, data1',
+      greetingInData2: 'Hello, data2',
+    }
+  },
+  computed: {
+    greetingInComputed() {
+      return 'Hello, computed property'
+    },
+    computedData1() {
+      return this.greetingInData1
+    },
+    computedGreetingInMethods() {
+      return this.greetingInMethods()
+    },
+  },
+  methods: {
+    greetingInMethods() {
+      return 'Hello, method'
+    },
+    returnData1() {
+      return this.greetingInData1
+    },
+    returnComputedData1() {
+      return this.computedData1
+    },
+  },
+})
+</script>

--- a/examples/app-vitest-full/pages/other/options-api.vue
+++ b/examples/app-vitest-full/pages/other/options-api.vue
@@ -1,14 +1,32 @@
 <template>
   <ul>
-    <li data-testid="greetingInSetup">{{ greetingInSetup }}</li>
-    <li data-testid="greetingInData1">{{ greetingInData1 }}</li>
-    <li data-testid="greetingInData2">{{ greetingInData2 }}</li>
-    <li data-testid="greetingInComputed">{{ greetingInComputed }}</li>
-    <li data-testid="computedData1">{{ computedData1 }}</li>
-    <li data-testid="computedGreetingInMethods">{{ computedGreetingInMethods }}</li>
-    <li data-testid="greetingInMethods">{{ greetingInMethods() }}</li>
-    <li data-testid="returnData1">{{ returnData1() }}</li>
-    <li data-testid="returnComputedData1">{{ returnComputedData1() }}</li>
+    <li data-testid="greetingInSetup">
+      {{ greetingInSetup }}
+    </li>
+    <li data-testid="greetingInData1">
+      {{ greetingInData1 }}
+    </li>
+    <li data-testid="greetingInData2">
+      {{ greetingInData2 }}
+    </li>
+    <li data-testid="greetingInComputed">
+      {{ greetingInComputed }}
+    </li>
+    <li data-testid="computedData1">
+      {{ computedData1 }}
+    </li>
+    <li data-testid="computedGreetingInMethods">
+      {{ computedGreetingInMethods }}
+    </li>
+    <li data-testid="greetingInMethods">
+      {{ greetingInMethods() }}
+    </li>
+    <li data-testid="returnData1">
+      {{ returnData1() }}
+    </li>
+    <li data-testid="returnComputedData1">
+      {{ returnComputedData1() }}
+    </li>
   </ul>
 </template>
 

--- a/examples/app-vitest-full/tests/nuxt/mount-suspended.spec.ts
+++ b/examples/app-vitest-full/tests/nuxt/mount-suspended.spec.ts
@@ -13,6 +13,7 @@ import ExportDefaultComponent from '~/components/ExportDefaultComponent.vue'
 import ExportDefineComponent from '~/components/ExportDefineComponent.vue'
 import ExportDefaultWithRenderComponent from '~/components/ExportDefaultWithRenderComponent.vue'
 import ExportDefaultReturnsRenderComponent from '~/components/ExportDefaultReturnsRenderComponent.vue'
+import OptionsApiPage from '~/pages/other/options-api.vue'
 
 import { BoundAttrs } from '#components'
 
@@ -78,6 +79,19 @@ describe('mountSuspended', () => {
     expect(component.find('div').html()).toMatchInlineSnapshot(
       '"<div>slot from mount suspense</div>"',
     )
+  })
+
+  it('should render asyncData and other options api properties within nuxt suspense', async () => {
+    const component = await mountSuspended(OptionsApiPage)
+    expect(component.find('[data-testid="greetingInSetup"]').text()).toBe('Hello, setup')
+    expect(component.find('[data-testid="greetingInData1"]').text()).toBe('Hello, data1')
+    expect(component.find('[data-testid="greetingInData2"]').text()).toBe('Hello, overwritten by asyncData')
+    expect(component.find('[data-testid="greetingInComputed"]').text()).toBe('Hello, computed property')
+    expect(component.find('[data-testid="computedData1"]').text()).toBe('Hello, data1')
+    expect(component.find('[data-testid="computedGreetingInMethods"]').text()).toBe('Hello, method')
+    expect(component.find('[data-testid="greetingInMethods"]').text()).toBe('Hello, method')
+    expect(component.find('[data-testid="returnData1"]').text()).toBe('Hello, data1')
+    expect(component.find('[data-testid="returnComputedData1"]').text()).toBe('Hello, data1')
   })
 
   it('can receive emitted events from components mounted within nuxt suspense', async () => {

--- a/examples/app-vitest-full/tests/nuxt/render-suspended.spec.ts
+++ b/examples/app-vitest-full/tests/nuxt/render-suspended.spec.ts
@@ -12,6 +12,7 @@ import ExportDefaultComponent from '~/components/ExportDefaultComponent.vue'
 import ExportDefineComponent from '~/components/ExportDefineComponent.vue'
 import ExportDefaultWithRenderComponent from '~/components/ExportDefaultWithRenderComponent.vue'
 import ExportDefaultReturnsRenderComponent from '~/components/ExportDefaultReturnsRenderComponent.vue'
+import OptionsApiPage from '~/pages/other/options-api.vue'
 
 import { BoundAttrs } from '#components'
 
@@ -85,6 +86,19 @@ describe('renderSuspended', () => {
       },
     })
     expect(screen.getByText(text)).toBeDefined()
+  })
+
+  it('should render asyncData and other options api properties within nuxt suspense', async () => {
+    const { getByTestId } = await renderSuspended(OptionsApiPage)
+    expect(getByTestId('greetingInSetup').textContent).toBe('Hello, setup')
+    expect(getByTestId('greetingInData1').textContent).toBe('Hello, data1')
+    expect(getByTestId('greetingInData2').textContent).toBe('Hello, overwritten by asyncData')
+    expect(getByTestId('greetingInComputed').textContent).toBe('Hello, computed property')
+    expect(getByTestId('computedData1').textContent).toBe('Hello, data1')
+    expect(getByTestId('computedGreetingInMethods').textContent).toBe('Hello, method')
+    expect(getByTestId('greetingInMethods').textContent).toBe('Hello, method')
+    expect(getByTestId('returnData1').textContent).toBe('Hello, data1')
+    expect(getByTestId('returnComputedData1').textContent).toBe('Hello, data1')
   })
 
   it('can receive emitted events from components rendered within nuxt suspense', async () => {

--- a/src/runtime-utils/mount.ts
+++ b/src/runtime-utils/mount.ts
@@ -59,7 +59,7 @@ export async function mountSuspended<T>(
   const vueApp = tryUseNuxtApp()?.vueApp
     // @ts-expect-error untyped global __unctx__
     || globalThis.__unctx__.get('nuxt-app').tryUse().vueApp
-  const { render, setup } = component as DefineComponent<Record<string, unknown>, Record<string, unknown>>
+  const { render, setup, data, computed, methods } = component as DefineComponent<Record<string, unknown>, Record<string, unknown>>
 
   let setupContext: SetupContext
   let setupState: Record<string, unknown>
@@ -116,6 +116,14 @@ export async function mountSuspended<T>(
                         ...component,
                         render: render
                           ? function (this: unknown, _ctx: Record<string, unknown>, ...args: unknown[]) {
+                            // Set before setupState set to allow asyncData to overwrite data
+                            if (data && typeof data === 'function') {
+                              // @ts-expect-error error TS2554: Expected 1 arguments, but got 0
+                              const dataObject: Record<string, unknown> = data()
+                              for (const key in dataObject) {
+                                renderContext[key] = dataObject[key]
+                              }
+                            }
                             for (const key in setupState || {}) {
                               renderContext[key] = isReadonly(setupState[key]) ? unref(setupState[key]) : setupState[key]
                             }
@@ -124,6 +132,17 @@ export async function mountSuspended<T>(
                             }
                             for (const key in passedProps || {}) {
                               renderContext[key] = passedProps[key]
+                            }
+                            if (methods && typeof methods === 'object') {
+                              for (const key in methods) {
+                                renderContext[key] = methods[key]
+                              }
+                            }
+                            if (computed && typeof computed === 'object') {
+                              for (const key in computed) {
+                                // @ts-expect-error error TS2339: Property 'call' does not exist on type 'ComputedGetter<any> | WritableComputedOptions<any, any>'
+                                renderContext[key] = computed[key].call(renderContext)
+                              }
                             }
                             return render.call(this, renderContext, ...args)
                           }

--- a/src/runtime-utils/render.ts
+++ b/src/runtime-utils/render.ts
@@ -66,7 +66,7 @@ export async function renderSuspended<T>(
   const vueApp = tryUseNuxtApp()?.vueApp
     // @ts-expect-error untyped global __unctx__
     || globalThis.__unctx__.get('nuxt-app').tryUse().vueApp
-  const { render, setup } = component as DefineComponent<Record<string, unknown>, Record<string, unknown>>
+  const { render, setup, data, computed, methods } = component as DefineComponent<Record<string, unknown>, Record<string, unknown>>
 
   let setupContext: SetupContext
   let setupState: SetupState
@@ -139,13 +139,13 @@ export async function renderSuspended<T>(
                         render: render
                           ? function (this: unknown, _ctx: Record<string, unknown>, ...args: unknown[]) {
                             // Set before setupState set to allow asyncData to overwrite data
-                            // if (data && typeof data === 'function') {
-                            //   // @ts-expect-error error TS2554: Expected 1 arguments, but got 0
-                            //   const dataObject: Record<string, unknown> = data()
-                            //   for (const key in dataObject) {
-                            //     renderContext[key] = dataObject[key]
-                            //   }
-                            // }
+                            if (data && typeof data === 'function') {
+                              // @ts-expect-error error TS2554: Expected 1 arguments, but got 0
+                              const dataObject: Record<string, unknown> = data()
+                              for (const key in dataObject) {
+                                renderContext[key] = dataObject[key]
+                              }
+                            }
                             for (const key in setupState || {}) {
                               renderContext[key] = isReadonly(setupState[key]) ? unref(setupState[key]) : setupState[key]
                             }
@@ -155,17 +155,17 @@ export async function renderSuspended<T>(
                             for (const key in passedProps || {}) {
                               renderContext[key] = passedProps[key]
                             }
-                            // if (methods && typeof methods === 'object') {
-                            //   for (const key in methods) {
-                            //     renderContext[key] = methods[key]
-                            //   }
-                            // }
-                            // if (computed && typeof computed === 'object') {
-                            //   for (const key in computed) {
-                            //     // @ts-expect-error error TS2339: Property 'call' does not exist on type 'ComputedGetter<any> | WritableComputedOptions<any, any>'
-                            //     renderContext[key] = computed[key].call(renderContext)
-                            //   }
-                            // }
+                            if (methods && typeof methods === 'object') {
+                              for (const key in methods) {
+                                renderContext[key] = methods[key]
+                              }
+                            }
+                            if (computed && typeof computed === 'object') {
+                              for (const key in computed) {
+                                // @ts-expect-error error TS2339: Property 'call' does not exist on type 'ComputedGetter<any> | WritableComputedOptions<any, any>'
+                                renderContext[key] = computed[key].call(renderContext)
+                              }
+                            }
                             return render.call(this, renderContext, ...args)
                           }
                           : undefined,

--- a/src/runtime-utils/render.ts
+++ b/src/runtime-utils/render.ts
@@ -66,7 +66,7 @@ export async function renderSuspended<T>(
   const vueApp = tryUseNuxtApp()?.vueApp
     // @ts-expect-error untyped global __unctx__
     || globalThis.__unctx__.get('nuxt-app').tryUse().vueApp
-  const { render, setup } = component as DefineComponent<Record<string, unknown>, Record<string, unknown>>
+  const { render, setup, data, computed, methods } = component as DefineComponent<Record<string, unknown>, Record<string, unknown>>
 
   let setupContext: SetupContext
   let setupState: SetupState
@@ -138,6 +138,14 @@ export async function renderSuspended<T>(
                         ...component,
                         render: render
                           ? function (this: unknown, _ctx: Record<string, unknown>, ...args: unknown[]) {
+                            // Set before setupState set to allow asyncData to overwrite data
+                            if (data && typeof data === 'function') {
+                              // @ts-expect-error error TS2554: Expected 1 arguments, but got 0
+                              const dataObject: Record<string, unknown> = data()
+                              for (const key in dataObject) {
+                                renderContext[key] = dataObject[key]
+                              }
+                            }
                             for (const key in setupState || {}) {
                               renderContext[key] = isReadonly(setupState[key]) ? unref(setupState[key]) : setupState[key]
                             }
@@ -146,6 +154,17 @@ export async function renderSuspended<T>(
                             }
                             for (const key in passedProps || {}) {
                               renderContext[key] = passedProps[key]
+                            }
+                            if (methods && typeof methods === 'object') {
+                              for (const key in methods) {
+                                renderContext[key] = methods[key]
+                              }
+                            }
+                            if (computed && typeof computed === 'object') {
+                              for (const key in computed) {
+                                // @ts-expect-error error TS2339: Property 'call' does not exist on type 'ComputedGetter<any> | WritableComputedOptions<any, any>'
+                                renderContext[key] = computed[key].call(renderContext)
+                              }
                             }
                             return render.call(this, renderContext, ...args)
                           }

--- a/src/runtime-utils/render.ts
+++ b/src/runtime-utils/render.ts
@@ -66,7 +66,7 @@ export async function renderSuspended<T>(
   const vueApp = tryUseNuxtApp()?.vueApp
     // @ts-expect-error untyped global __unctx__
     || globalThis.__unctx__.get('nuxt-app').tryUse().vueApp
-  const { render, setup, data, computed, methods } = component as DefineComponent<Record<string, unknown>, Record<string, unknown>>
+  const { render, setup } = component as DefineComponent<Record<string, unknown>, Record<string, unknown>>
 
   let setupContext: SetupContext
   let setupState: SetupState
@@ -139,13 +139,13 @@ export async function renderSuspended<T>(
                         render: render
                           ? function (this: unknown, _ctx: Record<string, unknown>, ...args: unknown[]) {
                             // Set before setupState set to allow asyncData to overwrite data
-                            if (data && typeof data === 'function') {
-                              // @ts-expect-error error TS2554: Expected 1 arguments, but got 0
-                              const dataObject: Record<string, unknown> = data()
-                              for (const key in dataObject) {
-                                renderContext[key] = dataObject[key]
-                              }
-                            }
+                            // if (data && typeof data === 'function') {
+                            //   // @ts-expect-error error TS2554: Expected 1 arguments, but got 0
+                            //   const dataObject: Record<string, unknown> = data()
+                            //   for (const key in dataObject) {
+                            //     renderContext[key] = dataObject[key]
+                            //   }
+                            // }
                             for (const key in setupState || {}) {
                               renderContext[key] = isReadonly(setupState[key]) ? unref(setupState[key]) : setupState[key]
                             }
@@ -155,17 +155,17 @@ export async function renderSuspended<T>(
                             for (const key in passedProps || {}) {
                               renderContext[key] = passedProps[key]
                             }
-                            if (methods && typeof methods === 'object') {
-                              for (const key in methods) {
-                                renderContext[key] = methods[key]
-                              }
-                            }
-                            if (computed && typeof computed === 'object') {
-                              for (const key in computed) {
-                                // @ts-expect-error error TS2339: Property 'call' does not exist on type 'ComputedGetter<any> | WritableComputedOptions<any, any>'
-                                renderContext[key] = computed[key].call(renderContext)
-                              }
-                            }
+                            // if (methods && typeof methods === 'object') {
+                            //   for (const key in methods) {
+                            //     renderContext[key] = methods[key]
+                            //   }
+                            // }
+                            // if (computed && typeof computed === 'object') {
+                            //   for (const key in computed) {
+                            //     // @ts-expect-error error TS2339: Property 'call' does not exist on type 'ComputedGetter<any> | WritableComputedOptions<any, any>'
+                            //     renderContext[key] = computed[key].call(renderContext)
+                            //   }
+                            // }
                             return render.call(this, renderContext, ...args)
                           }
                           : undefined,

